patches/server/9999-Add-entity-tracking-view-distance.patch

```diff
@@ -0,0 +1,131 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MartijnMuijsers <martijnmuijsers@live.nl>
+Date: Sat, 7 Aug 2021 12:10:17 +0200
+Subject: [PATCH] Add entity tracking view distance
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index c334f29c69c1e6e3fe55cd6695e7df400cf36058..e0706d41c3d7cc6d3d5508f28c97a2df4f39ac81 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -706,8 +706,10 @@ public class PaperWorldConfig {
+     }
+ 
+     public int noTickViewDistance;
++    public int entityTrackingViewDistance; // Paper - entity tracking view distance
```

Comment:
Concerning you are in a file introduced by paper, paper comments are, as far as I recall, not necessary here

Reply:
Thank you, I've removed the comments.